src/components/attribution-editor.js

```diff
@@ -168,7 +176,7 @@ function makeChange (path, onChange) {
 function renderFields (schema, metadata = {}, onChange) {
   let fields = []
   if (schema.title) {
-    fields.push(renderTextField('title', 'TITLE', metadata.title, onChange))
+    fields.push(renderTextField('title', 'TITLE', metadata.title.substr(0,metadata.title.indexOf('.')), onChange))
```

Comment:
This shoudn't be here, will screw up any title with `.` (ಠ_ಠ)  ... put it here instead https://github.com/the-grid/ed/blob/3dcd6344611a6738955d88b64260f1f18abdfac1/demo/demo.js#L67